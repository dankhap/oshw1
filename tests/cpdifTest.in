cp textFile.txt testOut.txt
diff textFile.txt testOut.txt
diff textFile.txt DifTextFile.txt
cp nFile no

