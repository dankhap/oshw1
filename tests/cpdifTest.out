textFile.txt has been copied to testOut.txt
0
1
No such file

